script.js

```diff
@@ -0,0 +1,462 @@
+            "", "", "", "", "", "", "", "", ""
+        ];
+
+        let rusButtons = [
+            "Ё","1", "2", "3", "4", "5", "6", "7", "8", "9", "0", "-", "=", "backspace", "tab",
+            "Й", "Ц", "У", "К", "Е", "Н", "Г", "Ш", "Щ", "З", "Х", "Ъ",'\\',
+            "CapsLock", "Ф", "Ы", "В", "А", "П", "Р", "О", "Л", "Д", "Ж", "Э", "enter",
+            "shift", "Я", "Ч", "С", "М", "И", "Т", "Ь", "Б", "Ю", ".", "↑","shift",
+            "ctrl", "win", "alt", " ", "alt", "ctrl", "←", "↓", "→"
+        ];
+
+        let rusShiftButtons = [
+            "Ё", "!", '"', "№", ";", "%", ":", "?", "*", "(", ")", "_", "+", "backspace", "tab",
+            "Й", "Ц", "У", "К", "Е", "Н", "Г", "Ш", "Щ", "З", "Х", "Ъ",'/',
+            "CapsLock", "Ф", "Ы", "В", "А", "П", "Р", "О", "Л", "Д", "Ж",  'Э', "enter",
+            "shift", "Я", "Ч", "С", "М", "И", "Т", "Ь", "Б", "Ю", ",", "↑","shift",
+            "ctrl", "win", "alt", " ", "alt", "ctrl", "←", "↓", "→"
+        ];
+
+        let rusShiftButtonsPic = [
+            "", '!', '"', "№", ";", "%", ":", "?", "*", "(", ")", "_", "+", "", "",
+            "", "", "", "", "", "", "", "", "", "", "", "", '/',
+            "", "", "", "", "", "", "", "", "", "", "",  '', "",
+            "", "", "", "", "", "", "", "", "", "", ",", "","",
+            "", "", "", "", "", "", "", "", ""
+        ];
+
+        let buttonsCode = [
+            "Backquote", "Digit1", "Digit2", "Digit3", "Digit4", "Digit5", "Digit6", "Digit7", "Digit8", "Digit9", "Digit0", "Minus", "Equal", "Backspace", "Tab",
+            "KeyQ", "KeyW", "KeyE", "KeyR", "KeyT", "KeyY", "KeyU", "KeyI", "KeyO", "KeyP", "BracketLeft", "BracketRight", "Backslash",
+            "CapsLock", "KeyA", "KeyS", "KeyD", "KeyF", "KeyG", "KeyH", "KeyJ", "KeyK", "KeyL", "Semicolon", "Quote", "Enter",
+            "ShiftLeft", "KeyZ", "KeyX", "KeyC", "KeyV", "KeyB", "KeyN", "KeyM", "Comma", "Period", "Slash", "topArr","ShiftRight",
+            "ControlLeft", "MetaLeft", "AltLeft", "Space", "AltRight", "ControlRight", "leftArr", "botArr",  "rightArr"
+        ];
+
+        let elementWhich = [
+            "192", "49", "50", "51", "52", "53", "54", "55", "56", "57", "48", "189", "187", "8", "9",
+            "81", "87", "69", "82", "84", "89", "85", "73", "79", "80", "219", "221","220",
+            "20", "65", "83", "68", "70", "71", "72", "74", "75", "76", "186", "222", "13",
+            "16", "90", "88", "67", "86", "66", "78", "77", "188", "190", "191", "38","16",
+            "17", "91", "18", "32", "18", "17", "37", "40",  "39"
+        ];
+
+        let j = 0;
+
+        if (localStorage.getItem('ru') == null) {
+            localStorage.setItem('ru', 'false');
+        }
+        document.addEventListener('keydown', () => {
+            if (event.altKey && event.shiftKey) {
+                this.properties.ru == 'true' ? this.properties.ru = 'false' : this.properties.ru = 'true';
```

Comment:
Then it should be "false", because if shift and alt are pressed in the same time there is a language change. I've set them as a string, because i've used LocalStorage to save the information about language, so to get the value of this property, i've used LocalStorage.getItem(), and the parametr is a string.